src/Controller/UsersController.php

```diff
@@ -437,24 +437,69 @@ public function deleteUser(Request $request, PDO $db)
      * @throws Exception
      */
     public function setTrusted(Request $request, PDO $db)
+    {
+        $this->setAdminOnlyBooleanField($request, $db, 'trusted');
+    }
+
+    /**
+     * Allow users to be set as admin
+     *
+     * @param $request Request
+     * @param $db      PDO
+     *
+     * @throws Exception
+     */
+    public function setAdmin(Request $request, PDO $db)
```

Comment:
Since the code requires php 7.3, this method can declare a void return type
```suggestion
    public function setAdmin(Request $request, PDO $db): void
```